packages/jupyter-ai/jupyter_ai/chat_handlers/export.py

```diff
@@ -26,7 +26,8 @@ def chat_message_to_markdown(self, message):
             return ""
 
     # Multiple chat histories in separate files
-    def get_chat_filename(self, path="./chat_history.md"):
+    def get_chat_filename(self, file_name="chat_history.md"):
+        path = os.path.join(self.root_dir, file_name)
```

Comment:
This whole logic seems unnecessary and sub-optimal. Imagine the user having 10 chat histories, this code will try existence of all 10 files before splitting out the name for the new file. I think we can get rid of the whole method and just append a timestamp to the filename.

A further improvement to the export would be to let the user specify a filename to append to with a fallback to a default name with timestamp.

```
/export -o model-tuning-exploration
```

Reply:
Agreed; using a formatted timestamp would likely be more helpful for users here.

Reply:
@3coins @dlqqq All suggestions above have been implemented, though the suggested `/export -o <filename>` is replaced by the simpler `/export  <filename>`. The documentation has also been updated as shown below: 
![image](https://github.com/jupyterlab/jupyter-ai/assets/29005/c7d85550-649f-4eb4-a7c8-f6dd6070c75e)

1. Files are now saved in the Jupyter root directory
2. Removed old code to increment the file suffix as it was slow. 
3. Added timestamps to each chat history file
4. Added the option to out in a chosen file name, else default to `chat_history-<timestamp>.md`. 
5. Updated documentation in the chat interface section.